SPECS/azurelinux-rpm-macros/azurelinux-rpm-macros.spec

```
@@ -59,6 +60,9 @@ Provides:       rust-srpm-macros
 Obsoletes:      mariner-rpm-macros <= 2.0-25
 Provides:       mariner-rpm-macros = %{version}-%{release}
 
+Obsoletes:      grub2-rpm-macros <= 2.06
```

```suggestion
Obsoletes:      grub2-rpm-macros <= 2.06-19
```

---

I had this originally, it doesn't work. I believe it would need the full dist tag. Felt easier to just use the major version since its a completely different versioning scheme.

---

Ah, right. How about:
```suggestion
Obsoletes:      grub2-rpm-macros <= 2.06-19%{?dist}
```

I keep forgetting the dist tag is part of the release. I'm guessing the `Obsoletes: mariner-rpm-macros` might have the same issue.

---

Having said that, it may be even more accurate to do `2.06-19.azl3`, since we may be carrying this over to future versions.